static_sources/eateryAttributes.json

```diff
@@ -0,0 +1,67 @@
+{
+  "104-West": {
+    "reserveLink": "https://www.opentable.com/restaurant/profile/1080703"
+  },
+  "Bear-Necessities": {
+    "exceptions": [
+      "Mobile Order Only"
+    ],
+    "isGet": true
+  },
+  "Becker-House-Dining": {
+    "reserveLink": "https://www.opentable.com/restaurant/profile/1080700"
+  },
+  "Bug-Stop-Bagels": {
```

Comment:
i didn't know there was a Bug Stop Bagels on campus :O

Reply:
see I didn't either, but this is BSB's official slug from Cornell - I checked the Cornell Dining JSON just to be sure